app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionsViewModel.java

```diff
@@ -56,6 +60,8 @@ public void destroy() {
                     this.rooms = extractRooms(sessions);
                     this.stimes = extractStimes(sessions);
 
+                    notifyPropertyChanged(BR.loadingVisibility);
```

Comment:
Ah, I see.
Okay, I don't have strong opinion. So it's okay the way you have fixed 👍